pwiz_tools/Skyline/Model/Results/RatioValue.cs

```diff
@@ -81,7 +81,8 @@ public static RatioValue ValueOf(double? ratio)
 
         public override string ToString()
         {
-            return Ratio + (double.IsNaN(StdDev) ? "" : (" rdotp " + DotProduct));  // Not L10N
+            // ReSharper disable once LocalizableElement
+            return Ratio + (double.IsNaN(StdDev) ? "" : (" rdotp " + DotProduct));

```

Comment:
There's more than one string on this line. Doesn't that mean the "disable once" isn't going to work?